target/test-classes/testData.properties

```diff
@@ -1,13 +1,17 @@
-browser=firefox
-#chromedriver=webdriver.chrome.driver
-#chromeDriverLocation=D:\\Automation\\drivers\\chromedriver.exe
-#firfoxdriver=webdriver.firfox.driver
+activeBrowser=chrome
+#activeBrowser=HTMLUnitDriver
+chromedriver=webdriver.chrome.driver
+chromeDriverLocation=D:\\Automation\\drivers\\chromedriver.exe
+#firefoxdriver=webdriver.firefox.driver
 #firefoxDriverLocation=D:\\Automation\\drivers\\firefoxdriver.exe
+firefoxDriverVersion=
+chromeDriverVersion=74.0
 
-url=http://localhost:8081/UserMgmt/UserRegistration.html
-
-
+usermgmtAppURL=http://localhost:8081/UserMgmt/UserRegistration.html
+#url=
```

Comment:
remove this unused commented out line

Reply:
Actually its being used by "DoubleClickHandlingTest" at line no 46

Reply:
Sorry, url i am removing :)

Reply:
Done...pls check